src/Statement.php

```diff
@@ -174,10 +174,8 @@ public function sign($privateKeyFile, $privateKeyPass, $options = array()) {
                 if (! $cert) {
                     throw new \Exception('Unable to read certificate for x5c inclusion: ' . openssl_error_string());
                 }
-
-                if (! openssl_x509_export($cert, $x5c, true)) {
-                    throw new \Exception('Unable to export certificate for x5c inclusion: ' . openssl_error_string());
-                }
+                
```

Comment:
I've removed the check here because it seems like if $cert is valid then this line will always execute successfully.